utils/cdi-provider/src/main/java/io/smallrye/config/cdi/providers/ConfigSourceProvider.java

```diff
@@ -0,0 +1,43 @@
+package io.smallrye.config.cdi.providers;
+
+import java.lang.annotation.Annotation;
+import java.util.Map;
+import java.util.Set;
+
+import javax.enterprise.context.Dependent;
+import javax.enterprise.inject.Produces;
+import javax.enterprise.inject.spi.InjectionPoint;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+/**
+ * Making the Config sources available via CDI
+ * 
+ * @author <a href="mailto:phillip.kruger@redhat.com">Phillip Kruger</a>
+ */
```

Comment:
Would it be more efficient if `ConfigSourceProvider` and `ConfigSourceMapProvider` were in a single class?

Reply:
Yes ! I made the change.